lib/activerecord-bitemporal/bitemporal.rb

```diff
@@ -233,13 +244,16 @@ module Scope
         }
         scope :valid_in, -> (from: nil, to: nil) {
           ignore_valid_datetime
-            .tap { |relation| break relation.where(klass.arel_table[:valid_to].gteq(from.in_time_zone.to_datetime)) if from }
-            .tap { |relation| break relation.where(klass.arel_table[:valid_from].lteq(to.in_time_zone.to_datetime)) if to }
+            .tap { |relation| break relation.where_bind("valid_to", :gteq, from.in_time_zone.to_datetime) if from }
+            .tap { |relation| break relation.where_bind("valid_from", :lteq, to.in_time_zone.to_datetime) if to }
         }
         scope :valid_allin, -> (from: nil, to: nil) {
           ignore_valid_datetime
-            .tap { |relation| break relation.where(klass.arel_table[:valid_from].gteq(from.in_time_zone.to_datetime)) if from }
-            .tap { |relation| break relation.where(klass.arel_table[:valid_to].lteq(to.in_time_zone.to_datetime)) if to }
+            .tap { |relation| break relation.where_bind("valid_from", :gteq, from.in_time_zone.to_datetime) if from }
+            .tap { |relation| break relation.where_bind("valid_to", :lteq, to.in_time_zone.to_datetime) if to }
+        }
+        scope :where_bind, -> (attr_name, operator, value) {
```

Comment:
[IMHO] 
`where_bind` is a common name and may conflict.
If you use it only in gem, I guess it's safer to add prefix like `bitemporal_where_bind`.